### PR TITLE
Add test for round-trip isl discovery in special cases

### PR DESCRIPTION
**Pull request by rtretyak**

### CODE REVIEW

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/RoundTripIslSpec.groovy [hunk 1]
```diff
@@ -33,6 +37,62 @@ class RoundTripIslSpec extends HealthCheckSpecification {
...
+        }
+        assumeTrue("Wasn't able to find round-trip ISL with a-switch", isl != null)
+        bfd && northboundV2.setLinkBfd(isl)
+
+        when: "Port down event happens"
+        antiflap.portDown(isl.srcSwitch.dpId, isl.srcPort)
+        cleanupActions << { antiflap.portUp(isl.srcSwitch.dpId, isl.srcPort) }
+
+        then: "ISL changed status to FAILED"
+        Wrappers.wait(WAIT_OFFSET * 2) {
```
Comment (andriidovhan) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/RoundTripIslSpec.groovy:56]: Do we really need to wait `WAIT_OFFSET * 2`?

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/RoundTripIslSpec.groovy [hunk 2]
```diff
@@ -33,6 +37,62 @@ class RoundTripIslSpec extends HealthCheckSpecification {
...
+        }
+
+        when: "Port up event happens, but traffic goes only in one direction"
+        lockKeeper.removeFlows([isl.aswitch])
+        cleanupActions << { lockKeeper.addFlows([isl.aswitch]) }
+        cleanupActions.pop().call() //antiflap.portUp(isl.srcSwitch.dpId, isl.srcPort)
+
+        then: "ISL is not getting discovered"
+        TimeUnit.SECONDS.sleep(discoveryInterval + 2)
+        Wrappers.wait(WAIT_OFFSET) {
```
Comment (andriidovhan) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/RoundTripIslSpec.groovy:68]: `wait` is needless here, I guess
The isl is already failed (L57-58)

**Review by andriidovhan — APPROVED** [review]

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/RoundTripIslSpec.groovy (hunk 1 above)
Comment (rtretyak) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/RoundTripIslSpec.groovy]: fixed

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/RoundTripIslSpec.groovy (hunk 2 above)
Comment (rtretyak) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/RoundTripIslSpec.groovy]: fixed

**Pull request merged**